class.taxonomy-single-term.php

```diff
@@ -112,18 +112,28 @@ class Taxonomy_Single_Term {
 	 * @var boolean
 	 */
 	protected $allow_new_terms = false;
+	
+	/**
+	 * Default for the selector
+	 * @since 0.2.2
+	 * @var array
+	 */
+	protected $default = [];
```

Comment:
this lib supports same version of php as WordPress, so shorthand notation won't work.